.github/workflows/push_pontis_publisher.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
           DOCKER_ACCESS_TOKEN: ${{ secrets.DOCKER_ACCESS_TOKEN }}
         run: |
           export $(grep -v '^#' .env | xargs)
-          export PONTIS_PACKAGE_VERSION=$(grep 'version' pontis-package/setup.cfg | grep -e '[0-9]\.[0-9]\.[0-9]' -o)
+          export PONTIS_PACKAGE_VERSION=$(grep 'version' pontis-package/setup.cfg | grep -e '[0-9]*\.[0-9]*\.[0-9]*' -o)
```

Comment:
AH good for catching the asterisks! When I started today review at the first commit from last week I was like "oh shoot I forgot to say that '[0-9]\.[0-9]\.[0-9]' would break for 0.4.12. Really good catch and my bad on missing this. Nice.